crates/daphne/src/protocol/aggregator.rs

```diff
@@ -352,11 +365,16 @@ impl DapTaskConfig {
                                 TransitionVar::Continued(outbound)
                             }
 
-                            Err(e @ (VdafError::Codec(..) | VdafError::Vdaf(..))) => {
+                            Err(e @ (VdafError::CodecDraft09(..) | VdafError::VdafDraft09(..))) => {
                                 tracing::warn!(error = ?e, "rejecting report");
                                 TransitionVar::Failed(TransitionFailure::VdafPrepError)
                             }
 
+                            Err(e @ (VdafError::Codec(..) | VdafError::Vdaf(..))) => {
+                                tracing::warn!(error = ?e, "rejecting report - latest");
+                                TransitionVar::Failed(TransitionFailure::VdafPrepError)
+                            }
```

Comment:
I don't think we need to distinguish between latest and draft 09 here.

Likewise below.